src/databricks/labs/remorph/reconcile/query_builder/expression_generator.py

```diff
@@ -144,7 +146,7 @@ def transform_expression(
     return expr
 
 
-def get_hash_transform(source: str):
+def get_hash_transform(source: Dialects):
```

Comment:
change it to DialectType: `from sqlglot.dialects.dialect import DialectType`

Reply:
done

Reply:
Same as above itshould be `Dialect` refer `config.py`